src/util/time.js

```diff
@@ -140,16 +117,23 @@ function computeDate (isoDate, duration) {
 }
 
 /**
- * Round the date, by default to the nearest minute
+ * Round the date, by default to the inferior date UTC with 0h0m0s
```

Comment:
This behavior is inconsistent and doesn’t match the name of the function. Why not round in all cases?

Reply:
Yes right, I'll change that.